config/charImage.py

```diff
@@ -38,6 +38,7 @@
 config.measurement.load(os.path.join(ObsConfigDir, "apertures.py"))
 config.measurement.load(os.path.join(ObsConfigDir, "kron.py"))
 config.measurement.load(os.path.join(ObsConfigDir, "convolvedFluxes.py"))
+config.measurement.load(os.path.join(ObsConfigDir, "gaap.py"))
```

Comment:
I don't think we  want to run gaap in charImage. charImage is not what makes the `src` catalogs, and I don't think we want to run it in `calibrate` (which does make the `src` catalogs) either. We don't run any galaxy measurements in single frame processing.